package.json

```diff
@@ -129,6 +132,7 @@
     "tslint-react": "3.5.1",
     "types-rlp": "0.0.1",
     "typescript": "2.8.1",
+    "u2f-api": "1.0.6",
```

Comment:
Do we need to add this dependency just for types? Or is there a way around that?

Reply:
Yeah, the package is written in TS so you have to have it as a dep to get the types.

Reply:
Ideally we can merge these types upstream so it won't have the extra package